test/cli-test.js

```diff
@@ -0,0 +1,196 @@
+//Sample test
+import assert from 'assert';
+import fs from 'fs';
+import fsExtra from 'fs-extra';
+
+// Tests need to import transpiled files that will be located in dist/ rather than src/
+var cli = require('../dist/cli');
+import constants from '../dist/constants/constants';
+
+const configs = {
+  projectName: 'jack',
+  description: 'Jack\'s Test Project',
+  version: '0.0.1',
+  author: 'Jack Meyer',
+  license: 'MIT',
+  ci: 'TravisCI',
+  container: 'Docker'
+};
+
+describe('Init Project', () => {
+  describe('#initProject()', () => {
+    it('should create a new directory with the project name', () => {
+
+
+      cli.initProject(configs);
+      assert.equal(true, fs.existsSync(configs.projectName));
+
+      // clean up directory that was created
+      fsExtra.removeSync(configs.projectName);
+    });
+  });
+});
+
+describe('Init Project', () => {
+  describe('#initProject()', () => {
+    it('should create a src directory underneath the project directory', () => {
+      cli.initProject(configs);
+      assert.equal(true, fs.existsSync(configs.projectName + '/src'));
+
+      // clean up directory that was created
+      fsExtra.removeSync(configs.projectName);
```

Comment:
This could be put in an after each block instead of calling it in every test

Reply:
Along that same line of thought, is there a way to put the `cli.initProject(configs);` in something to run before each block?

Reply:
I've made a change to add a before and after block.